test/lexer/regexp.ts

```diff
@@ -171,21 +173,19 @@ describe('Lexer - Regular expressions', () => {
       t.throws(() => scanSingleToken(state, context, 0));
     });
   }
-  fail('fails on /i/ii', '/ ', Context.AllowRegExp);
```

Comment:
There are lots of wrong message in this sections. And there are many duplicated tests on `/i/yy` and `/i/ss`.